src/classlibnative/bcltype/bignum.h

```diff
@@ -0,0 +1,140 @@
+            m_power10BigNumTable[4].m_blocks[1] = (UINT32)0x00000000;
+            m_power10BigNumTable[4].m_blocks[2] = (UINT32)0x00000000;
+            m_power10BigNumTable[4].m_blocks[3] = (UINT32)0x00000000;
+            m_power10BigNumTable[4].m_blocks[4] = (UINT32)0x2e953e01;
+            m_power10BigNumTable[4].m_blocks[5] = (UINT32)0x03df9909;
+            m_power10BigNumTable[4].m_blocks[6] = (UINT32)0x0f1538fd;
+            m_power10BigNumTable[4].m_blocks[7] = (UINT32)0x2374e42f;
+            m_power10BigNumTable[4].m_blocks[8] = (UINT32)0xd3cff5ec;
+            m_power10BigNumTable[4].m_blocks[9] = (UINT32)0xc404dc08;
+            m_power10BigNumTable[4].m_blocks[10] = (UINT32)0xbccdb0da;
+            m_power10BigNumTable[4].m_blocks[11] = (UINT32)0xa6337f19;
+            m_power10BigNumTable[4].m_blocks[12] = (UINT32)0xe91f2603;
+            m_power10BigNumTable[4].m_blocks[13] = (UINT32)0x0000024e;
+
+            // 10^256
+            m_power10BigNumTable[5].m_len = (UINT8)27;
+            m_power10BigNumTable[5].m_blocks[0] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[1] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[2] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[3] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[4] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[5] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[6] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[7] = (UINT32)0x00000000;
+            m_power10BigNumTable[5].m_blocks[8] = (UINT32)0x982e7c01;
+            m_power10BigNumTable[5].m_blocks[9] = (UINT32)0xbed3875b;
+            m_power10BigNumTable[5].m_blocks[10] = (UINT32)0xd8d99f72;
+            m_power10BigNumTable[5].m_blocks[11] = (UINT32)0x12152f87;
+            m_power10BigNumTable[5].m_blocks[12] = (UINT32)0x6bde50c6;
+            m_power10BigNumTable[5].m_blocks[13] = (UINT32)0xcf4a6e70;
+            m_power10BigNumTable[5].m_blocks[14] = (UINT32)0xd595d80f;
+            m_power10BigNumTable[5].m_blocks[15] = (UINT32)0x26b2716e;
+            m_power10BigNumTable[5].m_blocks[16] = (UINT32)0xadc666b0;
+            m_power10BigNumTable[5].m_blocks[17] = (UINT32)0x1d153624;
+            m_power10BigNumTable[5].m_blocks[18] = (UINT32)0x3c42d35a;
+            m_power10BigNumTable[5].m_blocks[19] = (UINT32)0x63ff540e;
+            m_power10BigNumTable[5].m_blocks[20] = (UINT32)0xcc5573c0;
+            m_power10BigNumTable[5].m_blocks[21] = (UINT32)0x65f9ef17;
+            m_power10BigNumTable[5].m_blocks[22] = (UINT32)0x55bc28f2;
+            m_power10BigNumTable[5].m_blocks[23] = (UINT32)0x80dcc7f7;
+            m_power10BigNumTable[5].m_blocks[24] = (UINT32)0xf46eeddc;
+            m_power10BigNumTable[5].m_blocks[25] = (UINT32)0x5fdcefce;
+            m_power10BigNumTable[5].m_blocks[26] = (UINT32)0x000553f7;
+        }
+    } m_initializer;
+
+    UINT8 m_len;
```

Comment:
You really should make it UINT32 rather than placing it after `m_blocks`. In general, the only advantage of using small integral types is storage size. In this case you can't actually save any storage due to alignment requirements. Instead your risk making the code larger because the compiler may need to insert widening conversion in some cases.